runtime/compiler/env/J9SharedCache.cpp

```diff
@@ -848,3 +862,92 @@ TR_J9SharedCache::getClassChainOffsetOfIdentifyingLoaderForClazzInSharedCache(TR
    uintptrj_t classChainOffsetInSharedCache = offsetInSharedCacheFromPointer(classChainIdentifyingLoaderForClazz);
    return classChainOffsetInSharedCache;
    }
+
+const void *
+TR_J9SharedCache::storeSharedData(J9VMThread *vmThread, char *key, J9SharedDataDescriptor *descriptor)
+   {
+   return _sharedCacheConfig->storeSharedData(
+         vmThread,
+         key,
+         strlen(key),
+         descriptor);
+   }
+
+#if defined(JITSERVER_SUPPORT)
+TR_J9JITServerSharedCache::TR_J9JITServerSharedCache(TR_J9VMBase *fe)
+   : TR_J9SharedCache(fe)
+   {
+   _stream = NULL;
+   }
+
+UDATA *
+TR_J9JITServerSharedCache::rememberClass(J9Class *clazz, bool create)
+   {
+   TR_ASSERT(_stream, "stream must be initialized by now");
+   auto clientData = TR::compInfoPT->getClientData();
+   PersistentUnorderedMap<J9Class *, UDATA *> & cache = clientData->getClassClainDataCache();
```

Comment:
Typo: `getClassClainDataCache` --> `getClassChainDataCache`. Possibly wrong in JITClientSession.hpp too.